server/db/fish.js

```diff
@@ -19,7 +19,15 @@ function createFish (
     }))
 }
 
+function updateFish (db, req, res) {
+  const fishId = Number(req.params.id)
+  return db('fishDetails')
+    .where('fishId', fishId)
+    .update(fishDetails => res.status(200).json(fishDetails))
+}
```

Comment:
These lines are the same as what is in your `server/routes/fish` file, that's more so where your `res.status...` belongs too.

Reply:
In which case this would look something similar to:
```
function editItem (ID, updatedItem, db =  connection) {
  return db('itemtable')
    .where('id', ID)
    .update({
      name: updatedItem.name,
      description: updatedItem.description,
    })
}
```